src/client/entity-editor/edition-section/reducer.ts

```diff
@@ -85,9 +85,9 @@ function reducer(
 			return state.set('depth', payload);
 		case UPDATE_WARN_IF_EDITION_GROUP_EXISTS:
 			if (!Array.isArray(payload) || !payload.length) {
-				return state.set('matchingNameEditionGroups', Immutable.List());
+				return state.set('matchingNameEditionGroups', []);
```

Comment:
I'm not sure but is it good idea to get rid of immutability here?
can't we just deserialize it in the component itself?

Reply:
The whole issue with the slowing down came from doing just that: https://github.com/metabrainz/bookbrainz-site/commit/7d02163d09246a27a159dbcdb3c6d54f60bfebb7#diff-2142be03733089165f1b3d5ce09055a7f3cd9a70a1559ac326acb9b8c57c26faL88

I didn't think it would have such a big impact, but obviously deserializing in the component is expensive (maybe even for an empty List).

I couldn't identify any upside in using a List over an array, so my original commit didn't make much sense in the first place.